admin/views/tool-import-export.php

```diff
@@ -102,7 +102,7 @@
 
 	<h2 class="nav-tab-wrapper" id="wpseo-tabs">
 		<?php foreach ( $tabs as $identifier => $tab ) : ?>
-			<a class="nav-tab" id="<?php echo $identifier; ?>-tab" href="#top#<?php echo $identifier; ?>"><?php echo $tab['label']; ?></a>
+			<a class="nav-tab" id="<?php echo esc_attr( $identifier . '-tab' ); ?>" href="<?php echo esc_url( '#top#' . $identifier ); ?>"><?php echo $tab['label']; ?></a>
```

Comment:
Expecting `<?php echo $tab['label']; ?>` to use `esc_html` as well.

Reply:
See my response: https://github.com/Yoast/wordpress-seo/pull/8378/files#r155085616